it/micrometer1.3/build.gradle

```diff
@@ -0,0 +1,12 @@
+dependencies {
+    [ 'micrometer-core', 'micrometer-registry-prometheus' ].each {
+        implementation("io.micrometer:$it") {
+            version {
+                // Will fail the build if the override doesn't work
+                strictly '1.3.9'
+            }
+        }
+    }
+}
+
+tasks.compileTestJava.source "${rootProject.projectDir}/core/src/test/java/com/linecorp/armeria/internal/common/metric/RequestMetricSupportTest.java"
```

Comment:
This test is better than nothing, but I wonder if it's not hard to collect all our `**MetricsIntegrationTest` for this check.

Reply:
Yeah, it's a good idea to cover them. Added.